include/NovelRT/Transform.h

```diff
@@ -16,42 +19,79 @@ namespace NovelRT {
     bool _active;
 
   public:
+    /**
+     * Creates a new Transform, with the specified position, rotation and scale.
+     * @param position The position.
+     * @param rotation The rotation.
+     * @param scale The scale.
+     */
     Transform(const Maths::GeoVector2<float>& position, float rotation, const Maths::GeoVector2<float>& scale) noexcept;
+    /**
+     * Creates a new Transform with the default values.
+     */
     Transform() noexcept;
 
+    /**
+     * Gets the Axis Aligned Bounding Box based on the position and scale of this transform.
+     */
     inline Maths::GeoBounds getAABB() const {
       auto scale = fmaxf(_scale.getX(), _scale.getY());
       return Maths::GeoBounds(_position, Maths::GeoVector2(scale, scale), 0);
     }
 
+    /**
+     * Gets the bounds of this transform.
+     */
     inline Maths::GeoBounds getBounds() const {
       return Maths::GeoBounds(_position, _scale, _rotation);
     }
 
+    /**
+     * Gets the constant position of this transform.
```

Comment:
Im unsure if "constant" is the right word here, but this is something we could discuss.

Reply:
Actually it's just because the whole method returned a `const`, and because for some awkward reason resharper does not like me to do `@copyDoc Transform::position`, I wanted to make it a bit... different? Anyway, `constant` is going away (and this wasn't an intended pun I swear)